concepticondata/conceptlists/Hantgan-2022-351.tsv

```diff
@@ -0,0 +1,352 @@
+Hantgan-2022-351-154	154	IRON	fer	621	IRON
+Hantgan-2022-351-155	155	JAW	mâchoire	798	JAW
+Hantgan-2022-351-156	156	JOINT	articulation	999	JOINT
+Hantgan-2022-351-157	157	KEY	clé	1472	KEY
+Hantgan-2022-351-158	158	KIDNEY	rate	489	KIDNEY
+Hantgan-2022-351-159	159	KILL	tuer	1417	KILL
+Hantgan-2022-351-160	160	KNEE	donner un coup de genou à [cf. kànjè].	1371	KNEE
+Hantgan-2022-351-161	161	KNEEL	(qn, chameau) s'agenouiller	66	KNEEL
+Hantgan-2022-351-162	162	KNIFE	couteau	1352	KNIFE
+Hantgan-2022-351-163	163	KNOW (SOMETHING)	connaître, savoir, reconnaître, remarquer	1410	KNOW (SOMETHING)
+Hantgan-2022-351-164	164	LADDER	écaille	1464	LADDER
+Hantgan-2022-351-165	165	LAUGH	le rire	1355	LAUGH
+Hantgan-2022-351-166	166	LEAF	feuille	628	LEAF
+Hantgan-2022-351-167	167	LEAVE	abandonner	1752	LEAVE
+Hantgan-2022-351-168	168	LEFT	(main etc.) gauche	244	LEFT
+Hantgan-2022-351-169	169	LICK	lécher	319	LICK
+Hantgan-2022-351-170	170	LIE DOWN	coucher, étendre, allonger, étaler	215	LIE DOWN
+Hantgan-2022-351-171	171	LIFT	amener, venir avec; amorcer (une matière, dans un discours) [irregular combination of kà ‘come’ plus ndù ‘with’; 3SgO kà:-.d-â, Imprt with 3SgO kà:-nd-â without -.suffix] [< kà 2].	501	LIFT
+Hantgan-2022-351-172	172	LIGHT (WEIGHT)	ê. faible, facile, à bon marché	1052	LIGHT (WEIGHT)
+Hantgan-2022-351-173	173	LIMP	boiter	366	LIMP
+Hantgan-2022-351-174	174	LION	lion	1386	LION
+Hantgan-2022-351-175	175	LOCK	couvercle; fermeture.	1596	LOCK
+Hantgan-2022-351-176	176	LOOK	examiner, regarder	1819	LOOK
+Hantgan-2022-351-177	177	LOOK FOR	chercher	1468	LOOK FOR
+Hantgan-2022-351-178	178	LOUSE	pou	1392	LOUSE
+Hantgan-2022-351-179	179	LUNCH	déjeuner	768	LUNCH
+Hantgan-2022-351-180	180	LUNG	poumon	688	LUNG
+Hantgan-2022-351-181	181	MAIZE	maïs	506	MAIZE
+Hantgan-2022-351-182	182	MAN	homme	1554	MAN
+Hantgan-2022-351-183	183	MARKET	marché	633	MARKET
+Hantgan-2022-351-184	184	MARROW	moelle	1835	MARROW
+Hantgan-2022-351-185	185	MARRY	(homme) épouser (femme)	203	MARRY
+Hantgan-2022-351-186	186	MAT	natte	195	MAT
+Hantgan-2022-351-187	187	MEAL	repas	1526	MEAL
+Hantgan-2022-351-188	188	MEAT	viande	634	MEAT
+Hantgan-2022-351-189	189	MEDICINE	médicament	499	CURE
+Hantgan-2022-351-190	190	MEET	(individus) se rassembler, se réunir	1831	MEET
+Hantgan-2022-351-191	191	MILK	lait	635	MILK
+Hantgan-2022-351-192	192	MILLET	mil	931	MILLET
+Hantgan-2022-351-193	193	MONEY	argent	1559	MONEY
+Hantgan-2022-351-194	194	MOON	lune	1313	MOON
+Hantgan-2022-351-195	195	MOSQUE	mosquéé	1603	MOSQUE
+Hantgan-2022-351-196	196	MOTHER	mère	1216	MOTHER
+Hantgan-2022-351-197	197	MOUSE	souris	1491	MOUSE
+Hantgan-2022-351-198	198	MOUTH	bouche	674	MOUTH
+Hantgan-2022-351-199	199	MUD	boue	640	MUD
+Hantgan-2022-351-200	200	NAME	nom	1405	NAME
+Hantgan-2022-351-201	201	NARROW	coincé	1267	NARROW
+Hantgan-2022-351-202	202	NEAR	proche	1942	NEAR
+Hantgan-2022-351-203	203	NECK	cou	1333	NECK
+Hantgan-2022-351-204	204	NEED (NOUN)	besoin	643	NEED (NOUN)
+Hantgan-2022-351-205	205	NEIGHBOR	voisin	643	NEED (NOUN)
```

Comment:
This is a double mapping from the previous. Also, _voisin_ appears below as well.